### ISSUE TITLE
long-term microwave Vegetation Optical Depth Climate Archive (VODCA)[Dataset Title/Name]: 

### ISSUE THREAD

**Issue by GonzaIrisarri** [samapriya/awesome-gee-community-datasets#207]
### Contact Details

gonzalo.irisarri@rothamsted.ac.uk

### Dataset description

The Global Long-term Microwave Vegetation Optical Depth Climate Archive VODCA
L. Moesinger, W. Dorigo, R. De Jeu, R. Van der Schalie, T. Scanlon, I. Teubner, and M. Forkel
Link to the paper: https://essd.copernicus.org/articles/12/177/2020/essd-12-177-2020.html

Link to the dataset: https://doi.org/10.5281/zenodo.2575599
Specifically, it would be good to have the following layer: https://zenodo.org/records/2575599/files/VODCA_Ku-band_1987-2017_v01.0.0.zip?download=1

Background: space-borne microwave radiometers have been providing measurements of radiation emitted by the Earth’s surface. From these measurements it is possible to derive vegetation optical depth (VOD), a model-based indicator related to the density, biomass, and water content of vegetation.

Many thanks,
Gonzalo.

### Earth Engine Snippet if dataset already in GEE

for example
```js
var asset_name = ee.ImageCollection("path to your collection")
```
Sample Code: Add a sample code maybe just adding your datasets in the code editor

### Enter license information

Creative Commons Attribution 4.0 International

### Keywords

vegetation optical depth (VOD)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by samapriya** [comment]
Are you sure this doesn't already exist here

https://gee-community-catalog.org/projects/vodca/?h=vod

**Comment by GonzaIrisarri** [comment]
Is the Ku-band uploaded? I couldn't see it.

Thank you very much for your help.

**Comment by samapriya** [comment]
Yeah the K band collection is in fact the KU band
var kband = ee.ImageCollection("projects/sat-io/open-datasets/VODCA/K-BAND")

**Comment by GonzaIrisarri** [comment]
Ah!, perfect 

I saw this: 
![image](https://github.com/samapriya/awesome-gee-community-datasets/assets/67291327/52ac5904-21f3-4509-ac3b-98760b2f0c70)

and wasn't sure about it. 

thank you

**Comment by samapriya** [comment]
Yeah the K band file is a single nc file as an example I can understand the confusion
